recipes/box2d/2.4.x/patches/0002-fix-shared-builds.patch

```diff
@@ -0,0 +1,10 @@
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -2,6 +2,7 @@ cmake_minimum_required(VERSION 3.8)
+ 
+ # https://cmake.org/cmake/help/latest/command/project.html
+ project(box2d VERSION 2.4.1)
++cmake_policy(SET CMP0077 NEW)
```

Comment:
This can be set in the recipe 

https://github.com/conan-io/conan-center-index/blob/e5973574dd62752e82d4290c44bc5d724a14a500/recipes/double-conversion/all/conanfile.py#L51-L52